.github/workflows/build.yml

```diff
@@ -41,14 +41,22 @@ jobs:
 
           - os: macos-10.15
             arch: "x86_64"
+            env:
+              MACOSX_DEPLOYMENT_TARGET: 10.9
           # not really needed separately, yet arm64 also creates a wheel for
           # CPython 3.8:
           #   https://cibuildwheel.readthedocs.io/en/stable/faq/#apple-silicon
           #   https://github.com/pypa/cibuildwheel/pull/704
           - os: macos-10.15
             arch: "arm64"
+            env:
+              CMAKE_OSX_ARCHITECTURES: "arm64"
+              MACOSX_DEPLOYMENT_TARGET: 11.0
           - os: macos-10.15
             arch: "universal2"
+            env:
+              CMAKE_OSX_ARCHITECTURES: "arm64;x86_64"
+              MACOSX_DEPLOYMENT_TARGET: 11.0
```

Comment:
This might need XCode 12.2+ / OSX 10.15.4+
- https://developer.apple.com/documentation/apple-silicon/building-a-universal-macos-binary
- https://developer.apple.com/documentation/xcode-release-notes/xcode-12_2-release-notes